### ISSUE TITLE
Allow sidewise scrolling/panning

### ISSUE THREAD

**Issue by johentsch** [TimeLineAnnotator/desktop#202]
**Desired behaviour**

Having zoomed into a timeline one needs a convenient way to pan right and left to navigate the piece. I would expect to scroll sidewise with a touchpad gesture, or using a common combination such as `Alt + [scroll]` or `Shift + [scroll]`.

**Current behaviour**

Vertical scroll results in zooming, just like horizontal scroll. `Alt + [scroll]` merely inverts the direction of zoom.

**Comment by FelipeDefensor** [comment]
Absolutely. @azfoo has pointed this out and implemented the scroll shortcuts. I'm planning to release this change together with an overhaul of the zooming tools, that will allow for things like fitting to screen and setting zoom percentages.

I believe the shortcuts can currently be enabled by uncommenting a single line of code. I can confirm that if you would like.

**Comment by azfoo** [comment]
If you comment out the original wheelEvent in tilia/ui/timelines/collection/view.py and replace it with wheelEvent_with_mod, you'll get the desired behaviour!
Again, as mentioned, that will likely come in another update
